### PR TITLE
idms-linux.org

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/idms-linux.org.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!--cdn. timed out-->
+<ruleset name="idms-linux.org">
+	<target host="idms-linux.org" />
+	<target host="www.idms-linux.org" />
+	<target host="wiki.idms-linux.org" />
```
Comment (J0WI) [inline src/chrome/content/rules/idms-linux.org.xml]: wiki redirects to http on other than /$
lists.idms-linux.org seems to work.

**Comment by galeksandrp** [comment]
Added.

**Comment by J0WI** [comment]
The http redirect problem is not yet fixed.

**Comment by galeksandrp** [comment]
Moved to comments.

**Comment by J0WI** [comment]
Now you also removed the comment for `cdn.idms-linux.org`

**Comment by galeksandrp** [comment]
Added back.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**